src/modules/reservation.js

```diff
@@ -0,0 +1,118 @@
+import { getReservation, addReservation } from './involvment.js';
+
+const setReservationHistory = (reservetionInfoArray, reservationUl) => {
+  reservationUl.innerHTML = '';
+  if (reservetionInfoArray.length) {
+    reservetionInfoArray.forEach((reservation) => {
+      const li = document.createElement('li');
+      li.classList.add('item');
+      li.innerHTML = `${reservation.date_start}  --  ${reservation.date_end}  By  ${reservation.username} `;
+      reservationUl.appendChild(li);
+    });
+  }
+};
+
+const displayMeals = (meal, reservetionInfoArray) => {
```

Comment:
Hi, @ghermaico135 
As an optional suggestion, you can use async here at the begging. That would make the code cleaner.